avocado/plugins/jobscripts.py

```diff
@@ -0,0 +1,69 @@
+import os
+import logging
+
+from avocado.utils import process
+from avocado.core.settings import settings
+from avocado.plugins.base import JobPre, JobPost
+
+
+CONFIG_SECTION = 'avocado.plugins.jobscripts'
```

Comment:
Currently we don't use `avocado` prefix, so IMO it should only be `plugins.jobscripts`.